### DISCUSSION TITLE
POI selections

### DISCUSSION THREAD

**Discussion by luigifab** [osmandapp/OsmAnd#19771] [Ideas]
Hello

Read the story:
- I would like to see the following POI: backery, supermarket, touristic place, drinkable water, toilets, campsite
- I select them and osmand display them
- ok!
- Now, I'm thirsty
- Near Aalborg there are a lot of POI displayed on the map
- So, I select only drinkable water, and osmand display them/it
- ok!
- I go to the POI... and no water! lol (water is off)
- Now I would like to see previous POI
- And this is the problem!

I must remember my previous selection 🤬
Because there is no bookmark system

Moreover I don't understand the sort order of the list, I never used "Commerce", it is displayed, I used "Toilets", it is not displayed!

When I use multiple selection, I can't search missing POI group.

Please do something.

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/31816829/a9020ecb-9909-46dc-88dc-493941c29a5e"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/31816829/4558385a-127a-4e67-a8eb-0d4467097342"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by luigifab** [comment]
Now, for example drinkable water has disappeared (for multiple selection), and I don't understand how to display it again!

**Reply by mankeil** [reply]
Have you tried searching for the POI using the search function? (magnifying glass icon on the map view)

**Reply by luigifab** [reply]
Yes, but after it still doesn't appear for multiple selection.

**Reply by DmitryAlexei** [reply]
@luigifab have you tried to save your search custom filters for future use?
https://osmand.net/docs/user/search/search-poi#save-new-custom-filters

**Reply by luigifab** [reply]
Wow, interesting, but I don't understand where is the functionality (ie create custom poi filter).

**Reply by DmitryAlexei** [reply]
You can find in the documentation a complete guide on how to create, edit and delete custom POI filters.
Please see 
https://osmand.net/docs/user/search/search-poi#custom-poi-search

**Reply by luigifab** [reply]
Yes, but I don't understand it.

I'm here, where is the "To save a custom filter, select everything you need, then: Tap the Show button and select the Save button for iOS, or the button with the icon indicating the save action for Android."

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/31816829/26f8f7ce-7b3d-4cd2-b333-4cfc7f551ff8"  width="300" height="auto"/> </td>
  </tr>
</table>

The only way to add multiple POI (for me) on the map, is here (sur couche et point d'intérêts via the world button of the previous screen, top left):

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/31816829/8b996f32-8c3d-4c84-8f9b-e328580e49c9"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by DmitryAlexei** [reply]
In the documentations should be "for Android: Tap the Show button, then tap on the filter icon, then tap 3 dots in the top-right corner of the screen and choose save filter. After that you can enter your name for the custom search filter" )) 
please see video

https://github.com/osmandapp/OsmAnd/assets/116079541/3eb214f6-7ad7-4e6c-9b32-04e9f446aa37

**Reply by luigifab** [reply]
Alright, at the end of the list! Understood.
